django_mongodb/client.py

```diff
@@ -4,11 +4,51 @@
 
 
 class DatabaseClient(BaseDatabaseClient):
-    executable_name = "mongo"
+    executable_name = "mongosh"
+    help_options = ("--help", "-h")
 
     @classmethod
     def settings_to_cmd_args_env(cls, settings_dict, parameters):
-        raise NotImplementedError
+        options = settings_dict.get("OPTIONS", {})
+        uri = options.get("uri")
```

Comment:
uri isn't a kwarg to MongoClient. If we're going to support this, it should be a separate discussion/issue.

Reply:
Yes, I put it in the options cause of that. I think this is nice to have for mongo's users.
But could be a separate discussion.

Reply:
Having understood the point :smile: , we could remove uri and let the users define it in the host parameter.